osu.Game/Screens/Edit/GameplayTest/EditorPlayer.cs

```diff
@@ -44,6 +46,25 @@ protected override void PrepareReplay()
 
         protected override bool CheckModsAllowFailure() => false; // never fail.
 
+        protected override IEnumerable<Container> GetCustomRulesetHotkeys() => new Container[]
+        {
+            new EditorHotkeyCurrentPositionExitContainer
+            {
+                Action = () =>
+                {
+                    editorState.Time = GameplayClockContainer.CurrentTime;
+                    PerformExit(false);
+                }
+            },
+            new EditorHotkeyQuickExitContainer
+            {
+                Action = () =>
+                {
+                    PerformExit(false);
+                }
+            }
```

Comment:
Why can't these just be implemented on the `EditorPlayer` class itself? I don't see any reason to nest like you have done here. Just implement `IKeyBindingHandler<GlobalAction>` here.